src/rgw/rgw_rados.cc

```diff
@@ -408,7 +408,7 @@ int RGWSystemMetaObj::init(CephContext *_cct, RGWRados *_store, bool setup_obj,
       r = read_id(name, id);
       if (r < 0) {
         if (r != -ENOENT) {
-          ldout(cct, 0) << "error in read_id for object name: " << name << " : " << cpp_strerror(-r) << dendl;
+          ldout(cct, 20) << "error in read_id for object name: " << name << " : " << cpp_strerror(-r) << dendl;
```

Comment:
the line `if (r != -ENOENT) {` was already added to prevent this from being logged in normal cases, so i think leaving this one at 0 is ok